### PR TITLE
Update to v1.6.1

### PR DESCRIPTION
**Pull request by Sad-Soul-Eater**
But firstly, the issue https://github.com/crowdsecurity/crowdsec/issues/2944 must be resolved

### CODE REVIEW

**Comment by Sad-Soul-Eater** [comment]
The docker image in the docker hub registry is up to date, should the chart be updated too?

File: charts/crowdsec/Chart.yaml [hunk 1]
```diff
@@ -87,9 +87,9 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.9.12
+version: 0.9.13
```
Comment (LaurenceJJones) [inline charts/crowdsec/Chart.yaml:90]: Please don't change the version we will do this before a release so please revert this change

**Review by LaurenceJJones — CHANGES_REQUESTED** [review]

File: charts/crowdsec/Chart.yaml (hunk 1 above)
Comment (Sad-Soul-Eater) [reply charts/crowdsec/Chart.yaml]: Done 👍

File: charts/crowdsec/Chart.yaml [hunk 2]
```diff
@@ -92,4 +92,4 @@ version: 0.9.12
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: v1.6.0
+appVersion: v1.6.1-1
```
Comment (LaurenceJJones) [inline charts/crowdsec/Chart.yaml:95]: We found a current issue could we bump the image to `1.6.1-2` hence why we do the rollout

**Review by LaurenceJJones — APPROVED** [review]
